Khan/wonder-blocks
# WB-556 docs(layout): Add docs for MediaLayout

## Summary

Added some examples for the MediaLayout component.

### Test plan

1. Open https://deploy-preview-469--wonder-blocks.netlify.com#medialayout
2. Check the new examples cover different uses for the MediaLayoutComponent.

<img width="962" alt="Screen Shot 2019-08-16 at 10 16 29 AM" src="https://user-images.githubusercontent.com/843075/63173950-f55b7e80-c00e-11e9-8a04-780912d06853.png">
